src/components/TagPicker/index.js

```diff
@@ -37,25 +37,27 @@ function TagPicker({selectedTag, tag, policyTags, policyRecentlyUsedTags, onSubm
         ];
     }, [selectedTag]);
 
+    const enabledTags = useMemo(() => {
+        if (!shouldShowDisabledAndSelectedOption) {
+            return policyTagList;
+        }
```

Comment:
As `policyTagList` is object, it will fail in `initialFocusedIndex` logic.

https://github.com/Expensify/App/assets/108292595/2dc0b86c-5c27-4210-8c57-634510960bb5

Reply:
Fixed!